test/timedemo_test.cpp

```diff
@@ -0,0 +1,65 @@
+#include <filesystem>
+#include <gtest/gtest.h>
+#include <iostream>
+
+#include "diablo.h"
+#include "engine/demomode.h"
+#include "options.h"
+#include "pfile.h"
+#include "utils/display.h"
+#include "utils/paths.h"
+
+using namespace devilution;
+
+namespace {
+
+bool Dummy_GetHeroInfo(_uiheroinfo *pInfo)
+{
+	return true;
+}
+
+void RunTimedemo(std::string timedemoFolderName)
+{
+	std::string unitTestFolderCompletePath = paths::BasePath() + "/test/fixtures/timedemo/" + timedemoFolderName;
+	paths::SetPrefPath(unitTestFolderCompletePath);
+	paths::SetConfigPath(unitTestFolderCompletePath);
+	LoadCoreArchives();
+	LoadGameArchives();
+
+	InitKeymapActions();
+	LoadOptions();
+
+	const int demoNumber = 0;
+
+	gbIsHellfire = false;
+	gbMusicOn = false;
+	gbSoundOn = false;
+	gbQuietMode = true;

```

Comment:
Maybe we should rename this to Headless?

Reply:
`gbHeadless`?
I think that would be a good idea. 🙂 
As part of this pr or separate?

Reply:
We can should actually drop the gb notation since we updated the code style.

I would prefer it here since it's getting used a not, and specifically to turn off the graphics

Reply:
No problem. I renamed it to `HeadlessMode`. 🙂